web-services/query/src/main/java/datawave/webservice/query/util/LookupUUIDUtil.java

```diff
@@ -418,10 +419,27 @@ public <T> T createUUIDQueryAndNext(final AbstractUUIDLookupCriteria unvalidated
         return response;
     }
     
-    private String getAuths(String logicName, String queryAuths, Principal principal) {
+    private Query createSettings(MultivaluedMap<String,String> queryParameters) {
+        Query query = responseObjectFactory.getQueryImpl();
+        query.setOptionalQueryParameters(queryParameters);
+        for (String key : queryParameters.keySet()) {
```

Comment:
I see that the mock returns emptySet for testing, but should we add a null check just in case?

Reply:
sure... ok hold on

Reply:
done